### ISSUE TITLE
Outer tank crossfeed doesn't seem to work

### ISSUE THREAD

**Issue by jwsteens** [flybywiresim/a32nx#6304]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-12-02T01:37:49+00:00",
  "ref": "master",
  "sha": "8dc5bc79bbbd69f4a90c20816c0ea5e22575cbad",
  "actor": "tracernz",
  "event_name": "manual"
}
```

### Describe the bug

I first came across this bug when my engines cut out mid-flight and my APU would not start either. At the moment I didn't realize what happened, but after some testing it seems like the fuel transferring from the outer tanks to the wing tanks disappears.

### Expected behavior

When the fuel from the outer tanks transfer to the wing tanks, you expect the same amount of fuel that gets substracted from the outer tank to be added to the wing tank.

### Steps to reproduce

1. Obtain external power (to view MCDU FUEL page)
2. Select FUEL page on MCDU
3. Start APU
4. Start defueling to ~1200kg
5. Observe how the fuel transferring from the external tanks does not go into the wing tanks or center tanks.

### References (optional)

![image](https://user-images.githubusercontent.com/74742373/144723509-f6ccae67-8f73-4ca9-96d9-74a5a4f9058f.png)

### Discord Username (optional)

willem#5819

**Comment by ChristianLutzCL** [comment]
I tested a bit what is going on here.

If your FOB is at 1035KG or higher, you "loose" fuel without any engine or APU running.
It isn't only disapperaring on the display, it is really disappearing.
OUTR TK XFRD with FOB below 1035KG works as expected.

[Watch here](https://youtu.be/PBwKhEY4UoI)

**Comment by donstim** [comment]
> I tested a bit what is going on here.
> 
> If your FOB is at 1035KG or higher, you "loose" fuel without any engine or APU running. It isn't only disapperaring on the display, it is really disappearing. OUTR TK XFRD with FOB below 1035KG works as expected.
> 
> [Watch here](https://youtu.be/PBwKhEY4UoI)
> 
> Edit: Not completely correct either. Seems to have something to do with the "defueling" function. Sporadically, the amount of fuel becomes less, even if the "defueling" process is already completed.

Thank you for the detailed report. I have also run some tests, and the system does not appear to be working as intended.  If you initially load enough fuel such that the outer tank transfer valves are closed (which is the normal initial loading condition and expected loading condition after fueling for a normal flight), the outer tank transfer should be working properly. So, I don't think this is an issue in a normal flight, contrary to the concern expressed in the initial issue write-up.

The problem appears to be an interaction with the de-fueling/re-fueling system perhaps not initializing properly. As an example, see the following video:  https://1drv.ms/u/s!Ajsy7Ey4_8uFz3p9bHtLELzlv2mI?e=cZZJaW. This is with batteries and external power on, APU and engines off. On initial load in, there is enough fuel in the inner tanks to not cause the outer tank transfer valve. Using the EFB to defuel to 1000 kg, the outer tank fuel transfer begins, but it does not go into the inner tanks. It disappears. However, selecting the refuel button again with the same 1000 kg requested fuel level, the outer tank fuel does transfer into the inner tank. 

Repeatedly selecting the 1000 kg fuel level does not reset the inner tank fuel level to zero. It retains the amount it had and fuel continues to transfer. Refueling to 4000 kg at this point, the fuel transfer continues to occur, even though there is more than 750 kg of fuel in each inner tank. (The transfer valves open when the inner tank fuel level decreases to about 750 kg.)

The way the system should work is that anytime the airplane is refueled (or defueled), the outer to inner fuel transfer valves should close. Whenever the inner tank fuel is less than or equal to 750 kg, the transfer valves should open. (If the refueling/defueling results in less than 751 kg of fuel in the inner tanks, the transfer valves should open pretty much immediately after the fueling process completes.) Whenever the transfer valves are open, the fuel level of the outer tanks should decrease, and the fuel level of the inner tanks should increase.

@Taz5150

**Comment by Taz5150** [comment]
> > I tested a bit what is going on here.
> > If your FOB is at 1035KG or higher, you "loose" fuel without any engine or APU running. It isn't only disapperaring on the display, it is really disappearing. OUTR TK XFRD with FOB below 1035KG works as expected.
> > [Watch here](https://youtu.be/PBwKhEY4UoI)
> > Edit: Not completely correct either. Seems to have something to do with the "defueling" function. Sporadically, the amount of fuel becomes less, even if the "defueling" process is already completed.
> 
> Thank you for the detailed report. I have also run some tests, and the system does not appear to be working as intended. However, if you initially load enough fuel such that the outer tank transfer valves are closed (which is the normal initial loading condition and expected loading condition after fueling for a normal flight), the outer tank transfer should be working properly. So, I don't think this is an issue in a normal flight, contrary to the concern expressed in the initial issue write-up.
> 
> The problem appears to be an interaction with the de-fueling/re-fueling system perhaps not initializing properly. As an example, see the following video: https://1drv.ms/u/s!Ajsy7Ey4_8uFz3p9bHtLELzlv2mI?e=cZZJaW. This is with batteries and external power on, APU and engines off. On initial load in, there is enough fuel in the inner tanks to not cause the outer tank transfer valve. Using the EFB to defuel to 1000 kg, the outer tank fuel transfer begins, but it does not go into the inner tanks. It disappears. However, selecting the refuel button again with the same 1000 kg requested fuel level, the outer tank fuel does transfer into the inner tank.
> 
> Repeatedly selecting the 1000 kg fuel level does not reset the inner tank fuel level to zero. It retains the amount it had and fuel continues to transfer. Refueling to 4000 kg at this point, the fuel transfer continues to occur, even though there is more than 750 kg of fuel in each inner tank. (The transfer valves open when the inner tank fuel level decreases to about 750 kg.)
> 
> The way the system should work is that anytime the airplane is refueled (or defueled), the outer to inner fuel transfer valves should close. Whenever the inner tank fuel is less than or equal to 750 kg, the transfer valves should open. (If the refueling/defueling results in less than 751 kg of fuel in the inner tanks, the transfer valves should open pretty much immediately after the fueling process completes.) Whenever the transfer valves are open, the fuel level of the outer tanks should decrease, and the fuel level of the inner tanks should increase.
> 
> @Taz5150

Thanks for the heads up. Will be taken care of with the new fuel system 👍🏻

**Comment by Taz5150** [comment]
Hi all. This issue is being solved and tested #7786.

Thank you for your contribution.
